### PR TITLE
Issue #PS-2281 feat: path change in players integration in teacher app

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

File: src/pages/play/content/[identifier].tsx [hunk 1]
```diff
@@ -108,10 +108,25 @@ const players: React.FC<SunbirdPlayerProps> = () => {
   return !loading ? <SunbirdPlayers player-config={playerConfig} /> : null;
 };
 
-export async function getStaticProps({ locale }: any) {
+export async function getStaticPaths() {
+  const identifiers = ['example1', 'example2'];
```
[the remark this answers is not in the document]
Comment (itsvick) [reply src/pages/play/content/[identifier].tsx:112]: which identifiers? @suvarnakale

File: src/pages/play/content/[identifier].tsx [hunk 2]
```diff
@@ -108,10 +108,25 @@ const players: React.FC<SunbirdPlayerProps> = () => {
   return !loading ? <SunbirdPlayers player-config={playerConfig} /> : null;
 };
 
-export async function getStaticProps({ locale }: any) {
+export async function getStaticPaths() {
+  const identifiers = [''];
+
+  const paths = identifiers?.map((id) => ({
```
Comment (itsvick) [inline src/pages/play/content/[identifier].tsx:114]: @suvarnakale this will always be empty array right?

Comment (suvarnakale) [reply src/pages/play/content/[identifier].tsx:114]: yes

Comment (suvarnakale) [reply src/pages/play/content/[identifier].tsx]: got it.. there is no need to map it.

**Pull request merged**